.github/workflows/treadmill-ci.yml

```diff
@@ -45,76 +41,78 @@ on:
         required: false
         type: string
         default: '["tests/c_hello.py"]' # Default to single test for backward compatibility
-
+      # New inputs for multi-board tests
+      multi-board:
+        required: false
+        type: string
+        default: 'false'
+      supervisor-id:
+        required: false
+        type: string
+        default: ''
+      board-descriptors:
+        required: false
+        type: string
+        default: ''
```

Comment:
Can we add descriptions on what these inputs are for, and their expected format?